shell/packages/sandstorm-db/db.js

```diff
@@ -2250,6 +2250,59 @@ if (Meteor.isServer) {
   };
 }
 
+if (Meteor.isServer) {
+  SandstormDb.prototype.deleteIdentity = function (identityId) {
+    this.removeApiTokens({ identityId: identityId });
```

Comment:
So, when we do this, what happens to:

* ApiTokens with `membraneRequirements` that reference that `identityId` (e.g. `permissionsHeld`)
* ApiTokens with `owner.user.identityId` referencing this identity

...and what do you expect the UI to show in these circumstances?